### DISCUSSION TITLE
Why icon is not rendered with llowing html in the option labels  ?

### DISCUSSION THREAD

**Discussion by PetroGromovo** [filamentphp/filament#10738] [Help]
Reading https://filamentphp.com/docs/3.x/forms/fields/select#allowing-html-in-the-option-labels docs I want to add icon and to options of select input
and I do it like :

```
    $optionText = 'Candidate';
    $svgHtml = '<svg class="heroicon-o-information-circle h-5 w-5 text-gray-400 dark:text-gray-500" </svg>';
    $optionsArray[$key] = '<span style="color:yellow" ' . $svgHtml . ' >' . $value . '</span>';

```
But I see option items with valid color but icon is rendered as "> " :

https://img001.prntscr.com/file/img001/hF5-2e55TWi7q2dGjw5KXg.png

I prefer to show different colors with classes like "heroicon-o-" which are used in filament...

How can I do it ?

"filament/filament": "^3.0-stable",
"filament/widgets": "^3.0-stable",
"laravel/framework": "^10.35.0",

Thanks in advance!

**Comment by leandrocfe** [comment]
`<svg class="heroicon-o-information-circle h-5 w-5 text-gray-400 dark:text-gray-500"/>`
?

**Reply by PetroGromovo** [reply]
No that did not help. Actually syntax was very similar my code.
